stdlib/REPL/src/REPL.jl

```diff
@@ -1418,6 +1418,7 @@ function out_transform(@nospecialize(x), n::Ref{Int})
 end
 
 function get_usings!(usings, ex)
+    ex isa Expr || return usings
     # get all `using` and `import` statements which are at the top level
     for (i, arg) in enumerate(ex.args)
```

Comment:
nit: `enumerate` is generally frowned upon when making a `Tuple{Any,Int}` iterator state
```suggestion
    for i in 1:length(ex.args)
        arg = ex.args[i]
```